x-pack/plugins/infra/public/common/alerts/use_summary_time_range.tsx

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { useMemo } from 'react';
+import {
```

Comment:
These imports are all from Observability. Would it make sense to create this hook in Observability and import it into Infra?

Reply:
Moved 👍